cmd/placement/main.go

```diff
@@ -49,7 +49,7 @@ func main() {
 	}
 
 	// Start Raft cluster.
-	raftServer := raft.New(cfg.raftID, cfg.raftInMemEnabled, cfg.raftBootStrap, cfg.raftPeers, cfg.raftLogStorePath)
+	raftServer := raft.New(cfg.raftID, cfg.raftInMemEnabled, cfg.raftPeers, cfg.raftLogStorePath)
```

Comment:
How does storing local logs work with the volumeless from scratch image?

Reply:
for volumeless(without persistent storage) image, it will work with inmemory storage.

Reply:
btw, scratch image cannot mount volume?